akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/EventSourcedBehavior.scala

```diff
@@ -81,6 +85,27 @@ object EventSourcedBehavior {
 
   }
 
+  /**
+   * The last sequence number that was persisted, can only be called from inside the handlers of an `EventSourcedBehavior`
+   */
+  def lastSequenceNumber(context: ActorContext[_]): Long = {
+    @tailrec
+    def extractConcreteBehavior(beh: Behavior[_]): Behavior[_] =
+      beh match {
+        case interceptor: InterceptorImpl[_, _] ⇒ extractConcreteBehavior(interceptor.nestedBehavior)
```

Comment:
I guess this works as long as someone else doesn't create their own wrapper style of behavior?

Reply:
If it hurts, don't do that!